grails-app/services/com/miniasaaslw/service/payment/PaymentService.groovy

```diff
@@ -46,7 +47,21 @@ class PaymentService {
         payment.save(failOnError: true)
     }
 
-    private Payment buildPaymentProperties(Payment payment, PaymentAdapter paymentAdapter){
+    public void updateToReceived(Long id) {
+        Payment payment = PaymentRepository.query([id: id]).get()
+
+        if (!payment) throw new RuntimeException("Cobrança não encontrada!")
+
+        Payment validatedPayment = validateUpdateToReceived(payment)
+        if (validatedPayment.hasErrors()) {
+            throw new ValidationException("Erro ao validar parâmetros da cobrança", validatedPayment.errors)
+        }
```

Comment:
```suggestion
        if (validatedPayment.hasErrors()) throw new ValidationException("Erro ao validar parâmetros da cobrança", validatedPayment.errors)
```
Quando for uma linha apenas, e dentro do { } tem um return ou um throw, que interrompem o codigo, podem fazer dessa maneira para deixar o codigo mais sucinto e rapido de ler